packages/features/bookings/lib/handleCancelBooking.ts

```diff
@@ -448,53 +448,79 @@ async function handler(input: CancelBookingInput) {
     allRemainingBookings
   );
 
-  const bookingToDeleteEventTypeMetadata = eventTypeMetaDataSchemaWithTypedApps.parse(
-    bookingToDelete.eventType?.metadata || null
-  );
+  try {
```

Comment:
Wrap a lot of post cancel booking operations in try/catch blocks so if one operation fails, it doesn't fail the whole cancel booking flow. Ex. if deleting a calendar event fails, then emails should still be sent out.